python/plugins/db_manager/db_plugins/oracle/connector.py

```diff
@@ -1307,6 +1307,14 @@ def renameSchema(self, schema, new_schema):
         # Unsupported in Oracle
         pass
 
+    def commentTable(self, schema, tablename, comment=None):
+        """Comment the table"""
+        return ''
+
+    def getComment(self, tablename, field):
```

Comment:
Move this method `getComment` into the base class `DBConnector` as a default implementation and override in PG connector only, instead of providing the same default implementation in all connectors but the PG one.